core/modules/hubotModule.js

```diff
@@ -0,0 +1,134 @@
+﻿/**
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Irregular whitespace not allowed](https://www.codacy.com/app/theprojectshop/Kassy/file/2530065980/issues/source?bid=3282987&fileBranchId=3282997#l1)